.github/workflows/bench.yml

```diff
@@ -107,6 +107,15 @@ jobs:
       - name: Install gzip
         run: sudo apt-get update && sudo apt-get install -y gzip ninja-build
 
+      - name: DuckDB extension build
```

Comment:
Do you know if this does, or can, use SCCACHE?

Reply:
@gatesn Note that we are caching the compiled binary as of today:
```
      - name: Cache duckdb compiled binary
        uses: runs-on/cache@v4
        id: cache-duckdb-binary
        with:
          fail-on-cache-miss: true
          key: "${{ runner.os }}-duckdb-linux_amd64-${{ env.DUCKDB_VERSION }}"
          path: ${{ github.workspace }}/duckdb/build/release/duckdb
```